client/landing/jetpack-cloud/sections/backups/rewind-flow/download.tsx

```diff
@@ -62,7 +62,7 @@ const BackupDownloadFlow: FunctionComponent< Props > = ( {
 			<h3 className="rewind-flow__title">{ translate( 'Create downloadable backup' ) }</h3>
 			<p className="rewind-flow__info">
 				{ translate(
-					'{{strong}}%(backupDisplayDate)s{{/strong}} is the selected point to create a download backup of. ',
+					'{{strong}}%(backupDisplayDate)s{{/strong}} is the selected point to create a downloadable backup. ',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.